saos-webapp/src/main/webapp/WEB-INF/view/common/javaScriptInitProd.jsp

```diff
@@ -5,13 +5,25 @@ contextPath = '${pageContext.request.contextPath}';
 
 -->
 </script>
```

Comment:
This script block does nothing. Consider extending comment to `<script>` tag. I would also change type of comment from `<!-- -->` to `<%-- --%>` to prevent from including it in rendered page (from the user point of view this comment is irrelevant)